src/components/PageBlocks/Configurator.vue

```diff
@@ -148,10 +170,20 @@ export default {
       return this.block.blockID === NoID
     },
   },
+
+  methods: {
+    updateRefresh (e) {
+      this.block.options.refreshRate = e.target.value < 5 && e.target.value > 0 ? 5 : e.target.value
```

Comment:
Either write this in multiple lines or add a comment so everyone knows what they are reading right away